tests/http-request-invalid/check.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+. ${TOPDIR}/util/functions.sh
+
+# Should have two alerts sid 2221034.
+n=$(jq_count eve.json 'select(.alert.signature_id == 2221034)')
```

Comment:
test conditions in test.yaml are preferred. Only if the logic can't be expressed there, a check script should be used.

Reply:
Ok, I got inspired by dnp3-dnp3_func-alert